content/runtime-config.md

```diff
@@ -122,6 +122,7 @@ Available rules:
     * **name** [String, required]: Matching job name.
     * **release** [String, required]: Matching release name.
 * **instance_groups** [Array of strings, optional]: Matches based on instance group names. Available in bosh-release v268+.
+* **lifecycle** [String, optional]: Matches based on lifecycle type, either `service` or `errand`. Available in bosh-release v262+.
```

Comment:
I think it might be worth to specify what a `service` is, i.e., anything that is not an `errand` VM, and that `colocated` errands may actually run on `service` VMs.